sopel/tests/factories.py

```diff
@@ -27,15 +43,13 @@ class BotFactory:
     def preloaded(
         self,
         settings: config.Config,
-        preloads: Optional[Iterable[str]] = None,
```

Comment:
Interesting that we had not imported `Optional` but weren't seeing an error? I guess we weren't type-checking here maybe? Anyway, I like the change.